pennylane/ops/op_math/composite.py

```diff
@@ -65,16 +65,16 @@ def __init__(
         if do_queue:
             self.queue()
 
-        self._set_batch_size()
+        self._batch_size = self._check_and_compute_batch_size()
 
-    def _set_batch_size(self):
+    def _check_and_compute_batch_size(self):
         batch_sizes = {op.batch_size for op in self if op.batch_size is not None}
         if len(batch_sizes) > 1:
             raise ValueError(
                 "Broadcasting was attempted but the broadcasted dimensions "
                 f"do not match: {batch_sizes}."
             )
-        self._batch_size = batch_sizes.pop() if batch_sizes else None
+        return batch_sizes.pop() if batch_sizes else None
```

Comment:
This is mostly a change of "style", right?

Reply:
Yep!

Reply:
Very nice :)